src/game/score/score.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import s from "./score.module.css";
+
+export function Score (props) {
+    let gameMode = (props.single ? 'SinglePlayer' : 'MultiPlayer');
+    let score = props.score;
```

Comment:
Looks like 'score' variable doesn't changes. Use const instead.

Reply:
or use destructurization
```suggestion
    export function Score ({score, single}) {
```

Reply:
Thx! I took into account both advices :)

```suggestion
export function Score ({single, score, onClick}) {
    const gameMode = single ? 'SinglePlayer' : 'MultiPlayer';
    
    return (
        <div className={styles.score__container}>
            <div className={styles.score__gametitle}>Game mode:</div>
            <div className={styles.score__gamemode}>{gameMode}</div>
            <div className={styles.score__scoretitle}>Score:</div>
            <div className={styles.score__scorevalue}>
                <span style={{color: 'red'}}>{score[0]} </span>:
                <span style={{color: 'blue'}}> {score[1]}</span>
            </div>
            <button className={styles.score__button} onClick={onClick}>Reset score</button>
        </div>
    );
}
```